.circleci/script/deploy_dags/master_dag.py

```diff
@@ -0,0 +1,81 @@
+import time
+from datetime import datetime
+from typing import Dict, List
+
+from airflow import DAG
+from airflow.models import DagRun, TaskInstance
+from airflow.operators.dummy import DummyOperator
+from airflow.operators.python import PythonOperator
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+from airflow.utils.session import create_session
+
+
+def get_report() -> None:
+    """Fetch dags run details and generate report"""
+    with create_session() as session:
+        last_13_dags_runs: List[DagRun] = (
+            session.query(DagRun).order_by(DagRun.execution_date.desc()).limit(13).all()
```

Comment:
We should not hardcode this limit

Reply:
Addressed it here https://github.com/astronomer/astronomer-providers/pull/136